bayer2ppm

```diff
@@ -234,9 +286,14 @@ def main():
     config.gammamode = argv.stringval("--gamma", default=None)
     config.verbose = not argv.exists("--quiet")
     config.debug = argv.exists("--debug")
+    config.process_yuv = argv.exists("--yuv")
+    config.outdir = argv.stringval("--outdir", default=None)
     show_help = argv.exists("--help")
     argv.exitIfAnyUnparsedOptions()
 
+    if config.verbose:
+        print('Bayer RAW convertor, ver %s\n' % config.version)
```

Comment:
Suggest changing printf-style syntax to Python3 f-strings:
`print(f"Bayer RAW converter, version {config.version}")`